bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/candidate/CandidateObsComparator.java

```diff
@@ -34,7 +34,7 @@ public int compare(Obs o1, Obs o2) {
 			return direction * ((ret != 0) ? ret : -o1.compareTo(o2));
 		}
 		
-		return direction * attr.getComparator().compare(o1, o2);
+		return direction * attr.getComparator(variant.getSchedule().getMiddlePoint()).compare(o1, o2);
```

Comment:
I've looked into `table` and `viewer` but couldn't find a more precise timestamp. I'll try passing the timestamp as a parameter to the constructor as we did with `CandidateObsComparator`. It's only instantiated once so it shouldn't be that much work.